HPXMLtoOpenStudio/measure.rb

```diff
@@ -1311,10 +1312,11 @@ def self.add_neighbors(runner, model, length)
   end
 
   def self.add_shading_schedule(runner, model, weather)
-    heating_season, @cooling_season = HVAC.get_default_heating_and_cooling_seasons(weather)
+    # Use BAHSP cooling season, and not year-round or user-specified cooling season, to ensure windows use appropriate interior shading factors
+    default_heating_season, @default_cooling_season = HVAC.get_default_heating_and_cooling_seasons(weather)
```

Comment:
Let's update these variables to be named more explicitly: "default_heating_months" and "default_cooling_months".